finetune/base.py

```diff
@@ -326,7 +326,7 @@ def _get_estimator_config(self):
             tf_random_seed=self.config.seed,
             save_summary_steps=self.config.val_interval,
             save_checkpoints_secs=None,
-            save_checkpoints_steps=None,
+            save_checkpoints_steps=10,
```

Comment:
Let's keep the config the same and specify this in the document_representation project?

Reply:
Why do we need this change at all? This reactivates the tensorflow default savers which we don't normally use.

Reply:
will change, especially know that I know about the cache_weights_to_file flag